docs/data-apps/funnel-builder/index.md

```diff
@@ -31,8 +31,14 @@ This data application provides an intuitive UI for building a funnel analysis an
 
 It works on any table that Snowplow’s Data Modeling User has access to, including the atomic events table and derived tables. It comes with some pre-built funnels based on out-of-the-box Snowplow events such as `page_views` and `link_clicks`, and you can save your own custom funnels to share with teammates.
 
-![Screenshot of a funnel configuration with built funnel charts.](./images/ecommerce-full-page.png)
 
+```mdx-code-block
+import ReactPlayer from 'react-player'
+import videoUrl from './images/funnel_builder_video.mp4'
+
+<ReactPlayer controls url={videoUrl} />
+<br/>
+```
```

Comment:
Any chance you could centre the window horizontally in the page?

Reply:
I've made them full width/height which makes it look better